src/components/PetTypesIndex.js

```diff
@@ -28,7 +28,9 @@ const PetTypesIndex = props => {
         <p>Here are the pet types we have available:</p>
       </div>
       <div className="row">
+        
```

Comment:
Any specific reasoning for these blank lines here? Spaces between codeblocks are specifically for above the return. This case its within the same div.